### PR TITLE
Reestructure project to colcon wp style.

### PR DESCRIPTION
**Pull request by lemunozm**
Rosis as a concon package style.

- Removed .repos
- Removed soss as submodule

A typical user of rosis will install rosis as a functional package (maybe in ros2 ecosystem or in his own workspace). This package will search the dependencies in the user workspace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,80 @@
-# rosis
-
+# ROSIS
 ROSIS (ROS2 Integration Services) is an implementation that uses [SOSS][soss] to connect ROS2 to different systems.
 
 ## Setup
+ROSIS uses a *colcon* to work.
```
Comment (Jaime-Burgos) [inline README.md]:
```suggestion
ROSIS uses *colcon* to work.
```

File: README.md [hunk 2]
```diff
@@ -1,48 +1,80 @@
...
+As dependencies, it needs a [SOSS][soss] environment with the `soss-ros2 and soss-mock` packages into it to work.
```
Comment (Jaime-Burgos) [inline README.md]:
```suggestion
As dependencies, it needs a [SOSS][soss] environment with the `soss-ros2` and `soss-mock` packages into it to work.
```

File: README.md [hunk 3]
```diff
@@ -1,48 +1,80 @@
...
 
-This repository has a workspace already organized in order to make easy starting a connection between ROS2 and any other system.
-
-It should be downloaded with the recursive option, in order to download SOSS as a third party. That way, the initial layout will include a folder with a workspace, which contains a `src` (source) directory, and inside that last folder, the SOSS project and the ROS2 system handle will be downloaded and ready to build.
+Clone this repository into [colcon workspace][colcon], and *source* the soss dependency:
```
Comment (Jaime-Burgos) [inline README.md]:
```suggestion
Clone this repository into a [colcon workspace][colcon], and *source* the soss dependency:
```

File: README.md [hunk 4]
```diff
@@ -1,48 +1,80 @@
...
+## Plugins packages
+ROSIS can connect ROS2 to any system that has a soss-plugin implemented.
+The official list of supported systems can be found at: [SOSS][soss].
 
-This will clone the system handles into the folder workspace/src/plugins.
+If you are a developer and you want to connect ROS2 to other systems,
+you may want to check our manual on [how to create a system handle](docs/CreatingSH.md)
 
 ## Usage
+* Into your colcon workspace, source the *colcon environment* in which ROSIS has been built (see [Setup](# Setup)):
```
Comment (Jaime-Burgos) [inline README.md]: The link was not working. I think it should work with the following change, based on [this answer](https://stackoverflow.com/a/6706594):

```suggestion
* Into your colcon workspace, source the *colcon environment* in which ROSIS has been built (see [Setup][]):
```

File: README.md [hunk 5]
```diff
@@ -1,48 +1,80 @@
...
+  ```
 
-The two systems will communicate through *rosis* now.
+* Source your own environment in order to allow rosis to find the plugins
+  ```
+  $ source install/local_setup.bash`
+  ```
 
-## Example - Connecting ROS2 with FIWARE
+* Run an instance of ROSIS with a [ROSIS configuration YAML file](#ROSIS configuration YAML file)
```
Comment (Jaime-Burgos) [inline README.md]: Same here

```suggestion
* Run an instance of ROSIS with a [ROSIS configuration YAML file][]
```

**Review by Jaime-Burgos — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lemunozm) [reply README.md]: Great!

File: README.md (hunk 4 above)
Comment (lemunozm) [reply README.md]: I did the change but still not working. It is weird. Maybe is because of the extra brackets.

Comment (lemunozm) [reply README.md]: Fixed: The problem was the space between `#` and `Setup`. And the Upper case `S`.

File: README.md (hunk 5 above)
Comment (lemunozm) [reply README.md]: Fixed: `#ROSIS configuration YAML file` by `#rosis-configuration-yaml-file`

**Review by Jaime-Burgos — APPROVED** [review]

**Pull request merged**